scripts/nginx/services.json

```diff
@@ -0,0 +1,25 @@
+{
+  "services": [
+    {
+      "name": "webclient",
+      "path": "/",
+      "port": 8080,
+      "type": "http",
+      "description": "Main web client"
+    },
+    {
+      "name": "buildertools",
+      "path": "/build/",
+      "port": 5001,
+      "type": "http",
+      "description": "Builder tools interface"
+    },
+    {
+      "name": "websockify",
+      "path": "/ws",
+      "port": 7901,
+      "type": "websocket",
+      "description": "WebSocket proxy for game connection"
+    }
+  ]
+}
```

Comment:
What is the advantage of services.json, over nginx.conf?

Reply:
> With a raw nginx.conf, to add a service one would need to know/remember the correct nginx syntax, remember to include the right headers for websockets vs regular HTTP, and hope you don't make any typos in the process. Very easy to make mistakes, especially if you're not familiar with nginx.

I'd certainly make all these mistakes, on my 1st attempt. It's good to have a less error-prone way. A different less-error-prone way would be to have pre-cooked nginx configs with placeholders for FQDNs, and a README that tells the user to drop any desired ones into /etc/nginx/sites-enabled/.